src/bindings/python/PyFormatMetadata.cpp

```diff
@@ -73,7 +81,11 @@ void bindPyFormatMetadata(py::module & m)
                 return ConstChildElementIterator(self);
             })
         .def("getChildElements", [](FormatMetadata & self) { return ChildElementIterator(self); })
-        .def("addChildElement", &FormatMetadata::addChildElement, "name"_a, "value"_a)
```

Comment:
I don't think this change is necessary. pybind11 handles `const char *` from Python string automatically. I have primarily been using `std::string` args in the constructors just for ease of checking for empty input.

Reply:
I'll revert that, but I also need to remove the test that check TypeError is raised when function is used with None for a parameter (segfault is generated)

Reply:
If you add `.none(false)` to the args which are causing segfaults when passing `None`, that should fix the issue. You can search for that in the bindings to see exmaples of where it has been added already.